react/react1/week2/todo-app/src/components/Heading.jsx

```diff
@@ -0,0 +1,16 @@
+import Subheading from "./Subheading";
+
+const Heading = ({todos}) => {
+    const count = todos.length;
+    let subheading;
+    const noun = count > 1 ? "tasks" : "task";
+    subheading = count + " " + noun;
```

Comment:
Why creating subheading and then re-assigning? And you can use string templating instead of concatenation